plugins/suggested-blocks/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "suggested-blocks",
```

Comment:
Can you add all of the fixes to the `package.json` that you had to make for the bitmap field? E.g. changing it to `@blockly/suggested-blocks` and `google/blockly-samples`?

Reply:
Done